app/views/layouts/adopter_foster_dashboard.html.erb

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<%= render "layouts/shared/head" %>
+
+  <body>
+    <%= render "layouts/shared/flash_messages" %>
+    <%= render 'layouts/shared/navbar' %>
+
+      <!-- Page Content -->
+      <main>
+        <section class="pt-5 pb-5">
+          <div class="container">
+            <div class="row align-items-center">
+              <!-- User info -->
+              <div class="col-xl-12 col-lg-12 col-md-12 col-12">
+                <!-- Bg -->
+                <div class="rounded-top" style="background: url('/assets/background/profile-bg.jpg') no-repeat; background-size: cover; height: 100px;"></div>
+                <div class="card px-4 pt-2 pb-4 shadow-sm rounded-top-0 rounded-bottom-0 rounded-bottom-md-2">
+                  <div class="d-flex align-items-end justify-content-between">
+                    <div class="d-flex align-items-center">
+                      <div class="me-2 position-relative d-flex justify-content-end align-items-end mt-n5">
+                        <%= render_avatar_partial(xl: true) %>
+                      </div>
+                      <div class="lh-1">
+                        <h2 class="mb-0"><%= "#{current_user.first_name} #{current_user.last_name}" %></h2>
+                      </div>
+                    </div>
+                    <div>
+                      <a href="add-course.html" class="btn btn-primary d-none d-md-block">Sample Button</a>
```

Comment:
Think we should make this href blank for now, since this is otherwise fetching a non-existent path.